### PR TITLE
Ui/move windows 3

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
Moving the "core" methods to `_legacy.c` feels like a pretty shitty solution, but seems like the easiest way to avoid a complete refactor.

File: src/openrct2/windows/_legacy.c [hunk 1]
```diff
@@ -133,3 +141,427 @@ void game_command_callback_pickup_staff(sint32 eax, sint32 ebx, sint32 ecx, sint
...
+{
+    audio_play_sound_at_location(
+        SOUND_PLACE_ITEM,
+        gCommandPosition.x,
+        gCommandPosition.y,
+        gCommandPosition.z
+    );
+
+    Ride *ride = get_ride(gRideEntranceExitPlaceRideIndex);
+    if (ride_are_all_possible_entrances_and_exits_built(ride)) {
```
Comment (Gymnasiast) [inline src/openrct2/windows/_legacy.c]: This whole file does not adhere to the code style.

Comment (janisozaur) [reply src/openrct2/windows/_legacy.c]: This is legacy **C** code just moved to a separate file. I'm fine with it staying as-is.

Comment (Gymnasiast) [reply src/openrct2/windows/_legacy.c]: And I am not. The code rules apply to everything. You touch code, you fix it.

**Pull request closed without merging**